psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -372,7 +372,7 @@ def scf_iterate(self, e_conv=None, d_conv=None):
                 # need to ensure orthogonal orbitals and set epsilon
                 status.append(base_name + "conv")
                 core.timer_on("HF: Form C")
-                self.form_C()
+                self.form_C(0.0)
```

Comment:
I would prefer the binding to be updated to know about the default parameter. When I see this function call I have no idea what 0.0 is supposed to mean.

[Pybind11 Default Arguments](https://pybind11.readthedocs.io/en/stable/basics.html#default-argumentsl)

Reply:
Thanks for the tip!